docs/ops/matrix/Einsum_7.md

```diff
@@ -0,0 +1,228 @@
+## Einsum <a name="Einsum"></a> {#openvino_docs_ops_matrix_Einsum_7}
+
+**Versioned name**: *Einsum-7*
+
+**Category**: Matrix multiplication
+
+**Short description**: *Einsum* performs the Einstein summation convention on the operands.
+
+**Detailed description**: *Einsum* can represent many common multidimensional linear algebraic tensor operations: matrix multiplication;
+inner (or dot), outer and cross products; transpose; trace and diagonal extraction.
+Also, the single *Einsum* operation can express complex combination of these common linear algebraic tensor operations on multiple operands,
```

Comment:
the single *Einsum* operation ---> a single *Einsum* operation

Reply:
Fixed.